workflows/standalone_modules/wf_kraken2_pe.wdl

```diff
@@ -23,25 +23,25 @@ workflow kraken2_pe_wf {
   }
   call krona_task.krona as krona {
     input:
-      kraken2_report = kraken2_pe.kraken2_report,
+      kraken2_report = kraken2.kraken2_report,
       samplename = samplename
   }
   call versioning.version_capture {
     input:
   }
   output {
     # PHBG Version Captures
-    String kraken2_pe_wf_version = version_capture.phb_version
-    String kraken2_pe_wf_analysis_date = version_capture.date
+    String kraken2_wf_version = version_capture.phb_version
```

Comment:
I think included the `_pe` here makes more sense than removing it. Could you it add it back?